public-api/src/main/java/org/pmiops/workbench/publicapi/CdrVersionsController.java

```diff
@@ -13,7 +13,6 @@
 import org.pmiops.workbench.db.model.CommonStorageEnums;
```

Comment:
Could you also remove this enum from the Swagger / enum converter utility?

Reply:
Done. Thanks.

Reply:
Was distracted working on something else. Pushed only the change from public-api.
Sorry.